tests/playground.spec.js

```diff
@@ -0,0 +1,252 @@
+const { test, expect } = require('@playwright/test')
+const { times } = require('lodash')
+
+const fs = require('fs')
+const path = require('path')
+
+test.beforeEach(async ({ page }) => {
+  const date = new Date(Date.UTC(1982, 3, 15))
+  await page.clock.setFixedTime(date)
+
+  // const environment = process.env.ENVIRONMENT
+  // const url = environment === 'prod'
+  //   ? 'https://cypress-playground.s3.eu-central-1.amazonaws.com/index.html'
+  //   : './src/index.html'
+  // await page.goto(url)
+  await page.goto('https://cypress-playground.s3.eu-central-1.amazonaws.com/index.html')
+})
+
+test.describe('Playwright Playground', () => {
+  test('shows a promotional banner', async ({ page }) => {
+    const banner = await page.locator('#promotional-banner')
+    expect(await banner).toBeVisible()
+    expect(await banner.innerText()).toContain('📣 Get to know the Cypress, from Zero to the Cloud course!')
+
+    const link = await banner.locator('a')
+    expect(await link.getAttribute('target')).toEqual('_blank')
```

Comment:
Can be replaced by `await expect(link).toHaveAttribute('target', '_blank')`

eslint-plugin-playwright auto-fix this one